adcm_aio_client/core/filters.py

```diff
@@ -53,7 +53,16 @@ def inline_filters_to_query(self: Self, filters: dict[str, FilterValue]) -> Quer
         converted_filters = deque()
 
         for inline_filter, value in filters.items():
-            attr, op = inline_filter.rsplit("__", maxsplit=1)
+            try:
+                attr, op = inline_filter.rsplit("__", maxsplit=1)
+            except ValueError:
+                message = (
+                    f"Invalid inline filter format: {inline_filter}. "
+                    "Attribute and operation should be joined with `__` for inline filters. "
+                    f"Maybe you've meant `{inline_filter}__eq={value}`"
+                )
+                raise FilterPreparationError(message) from None
```

Comment:
Maybe it's worth using one error for filters: `InvalidFilterError` ?

Reply:
agreed, since we don't use `FilterPreparationError` anywhere, I'll remove it